CMakeLists.txt

```diff
@@ -720,7 +720,7 @@ ELSE()
   MESSAGE(STATUS "==== OpenZWave not found, support disabled!")
 ENDIF(OpenZWave)
 
-IF(EXISTS /sys/class/gpio)
+IF(EXISTS /sys/class/gpio OR WITH_GPIO)
```

Comment:
Question, if someone uses -DWITH_GPIO=YES , it is already added to the definitions ? It seems with this change it is added twice ?